### ISSUE TITLE
Black blocks instead of context menu

### ISSUE THREAD

**Issue by ZapAnton** [LightTable/LightTable#2044]
On ubuntu 14.04 LTS, while using 0.8.0-alpha vesion of LT, i'm getting folowing black blocks instead of context menu, when I right-click in any area of LT : 

![lt_bug1](https://cloud.githubusercontent.com/assets/8658767/11225406/01a092c6-8d8b-11e5-8280-4a9be93d601c.png)

![lt_bug2](https://cloud.githubusercontent.com/assets/8658767/11225417/135adfbc-8d8b-11e5-8031-61b6ac44abd5.png)

I made no changes to user behaviors/keysmaps or plugins

The graphics driver is open Nouveau display driver

**Comment by kenny-evitt** [comment]
Can you try another driver? Maybe the proprietary one for your graphics card?

Would you also mind trying [Atom](https://atom.io/) and confirming whether you have the same or similar problems with it?

**Comment by tvanden** [comment]
Hey, been a few days on this. I was having this same issue. I did the troubleshooting you suggested. I first tried Atom, and there were no similar problems. All the menus work. 
I checked my driver, and I was running on xorg driver, so I changed to proprietary nvidea driver, and the menus are all now working. 
Hope that helps at all.

**Comment by kenny-evitt** [comment]
@tvanden Thanks! That seems to point to something having been fixed upstream in Electron, which both LT and Atom use.

**Comment by cldwalker** [comment]
There is [at least one known issue](https://github.com/atom/electron/issues/1277#issuecomment-84480408) with chromium and the nouveau driver. There isn't anything we can do for this if the underlying issue is Chromium via electron. If the next electron upgrade #2052  doesn't resolve this, I'm inclined to close this unless @tvanden is going to dig deeper into this

**Comment by cldwalker** [comment]
There is [at least one known issue](https://github.com/atom/electron/issues/1277#issuecomment-84480408) with chromium and the nouveau driver. There isn't anything we can do for this if the underlying issue is Chromium via electron. If the next electron upgrade #2052  doesn't resolve this, I'm inclined to close this unless @tvanden is going to dig deeper into this

**Comment by tvanden** [comment]
I'd like to help with this. I'm not sure what I can do to further troubleshoot. Got some tips on what I can try? @kenny-evitt @cldwalker

**Comment by cldwalker** [comment]
@tvanden Great! It'd probably help to [inspect the html](https://github.com/LightTable/LightTable/wiki/FAQ#how-do-i-inspect-any-html-element-in-light-table) of the black blocks to determine what part of LightTable is black. Since you're seeing this in the workspace menu, the relevant html is probably in the [workspace namespace](https://github.com/LightTable/LightTable/blob/c99c0b77240a2cfb740e518435df196eea307e1e/src/lt/objs/sidebar/workspace.cljs). It's also worth determining if black menus occur when you try different themes in your user.behaviors file or if you right click in a tab

**Comment by kenny-evitt** [comment]
@tvanden You can also try just running a build of the latest source version. We just upgraded the version of Electron and, given that you commented earlier that you weren't having this problem with Atom (which also uses Electron), this might be resolved now.

**Comment by tvanden** [comment]
@kenny-evitt Hey, I tried to install from source. Everything went smoothly, no errors. Other than a couple missing dependencies I had to get. However, it won't start. When I launch it, it opens up the window, shows the Light Table logo, and thats it. It never loads past there. There is no menu that loads, and it freezes. Can't close it. Any idea? Would I have to uninstall the packaged version before building from source?

**Comment by kenny-evitt** [comment]
@tvanden If you would, add a comment with the missing dependencies so we can update the developer guide.

As for the source build, how are you running it? I'd suggest just running the executable directly. It should be under *builds/lighttable-0.8.0-linux* and named *LightTable*.

**Comment by tvanden** [comment]
@kenny-evitt I meant for dependencies npm. You already have listed. I was just being dumb and tried to install it first. 

That is how I was trying to run the source build.

**Comment by kenny-evitt** [comment]
@tvanden Okay. But after you build source, how (exactly) are you running the source-build Light Table executable?

**Comment by tvanden** [comment]
I've tried both opening the builds folder in nautilus and double clicking
the LightTable Executable. also, In terminal, cd builds. cd
lighttable-0.8.0 then ./LightTable

**Comment by kenny-evitt** [comment]
@tvanden If you run it in a Terminal window do you see anything logged or output there when you run LT?

Are you running Ubuntu 14.04 too?

**Comment by tvanden** [comment]
No there is zero output. I am running Ubuntu 14.04 yes

**Comment by kenny-evitt** [comment]
@tvanden Would you post a screenshot?

**Comment by tvanden** [comment]
![ltscreenshot](https://cloud.githubusercontent.com/assets/7208221/11824992/ed963186-a339-11e5-8d00-dc1fe92ed401.png)

@kenny-evitt 
Here is a Screenshot of when I try to run it. It never gets beyond that point. Never shows any output or anything.

**Comment by tvanden** [comment]
@kenny-evitt 
I think this may be fixed?
I had to do a fresh install of ubuntu on an older laptop today, and I installed lighttable on it. And there was no issues with the right click menu. Looks like binary version 0.34.5 may have fixed this issue? I tested it out on my desktop as well, and sure enough on Nouveau driver version 0.31.1 I got the black blocks, but as soon as I downloaded the new file from your site and unpacked it, binary version 0.34.5, the issue was no longer showing up for me.
